src/util/minmax_heap.h

```diff
@@ -19,18 +19,20 @@ typedef struct heap {
   void* cmp_ctx;
   void** data;
   mmh_free_func free_func;
-} heap_t;
+} mm_heap_t;
 
-heap_t* mmh_init(mmh_cmp_func cmp, void* cmp_ctx, mmh_free_func free_func);
-heap_t* mmh_init_with_size(size_t size, mmh_cmp_func cmp, void* cmp_ctx, mmh_free_func free_func);
-void mmh_free(heap_t* h);
+mm_heap_t* mmh_init(mmh_cmp_func cmp, void* cmp_ctx, mmh_free_func free_func);
+mm_heap_t* mmh_init_with_size(size_t size, mmh_cmp_func cmp, void* cmp_ctx, mmh_free_func free_func);
+void mmh_free(mm_heap_t* h);
+void mmh_clear(mm_heap_t* h);
 
-void mmh_dump(heap_t* h);
-void mmh_insert(heap_t* h, void* value);
-void* mmh_pop_min(heap_t* h);
-void* mmh_pop_max(heap_t* h);
-void* mmh_peek_min(const heap_t* h);
-void* mmh_peek_max(const heap_t* h);
-void* mmh_exchange_min(heap_t* h, void* value); // combines pop-and-then-insert logic
+// void mmh_dump(mm_heap_t* h);
```

Comment:
unused?

Reply:
Yes. Implementation commented out for years